packages/database/src/migrations/20210621034818-FixSbAtLeastOneStaffMemberTrained-modifies-data.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboardIds = [
+  'UNFPA_Country_Facilities_offering_services_At_Least_1_Matrix_SB',
+  'UNFPA_Country_Facilities_offering_services_At_Least_1_Matrix_VU',
+];
+
+exports.up = async function (db) {
+  for (const dashboardId of dashboardIds) {
```

Comment:
Totally fine to do this the way you've done it, but it would be slightly more efficient to do an `id IN (${arrayToDbString(dashboardIds)})` rather than iterate through (probably negligible difference when you're only iterating through two dashboards)

Reply:
Thanks for sharing! I will use `id IN` next time.